src/validateFunctions.js

```diff
@@ -0,0 +1,9 @@
+export const createOutput = (error, parsedData, rawData) => {
+    const response =  {
+        valid: error.length > 0,
```

Comment:
This is incorrect.
If `error.length > 0 == true` then it's invalid.
If `error.length > 0 == false` then it's valid.
Either change the `>` to a `<`, or change the condition to `errors.length == 0`.

Reply:
Ups, you are right. Thanks!